bb/utils.clj

```diff
@@ -0,0 +1,6 @@
+(ns utils
+  (:require [babashka.process :refer [sh]]
+            [clojure.string :as str]))
+
+(defn latest-sha []
+  (str/trim (:out (sh "git rev-parse HEAD"))))
```

Comment:
My preference would be to inline this in `bb.edn` to not have another extra dir & file.

Reply:
This can lead two ways: big chunks of code in bb.edn, making it, as you pointed out once, less REPL friendly. Or we start right now with a separate file for developing larger chunks of code and move the code that is already in `:init` to this file.